src/components/ChatEntry.js

```diff
@@ -1,38 +1,35 @@
 import React from 'react';
-import { useState } from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
 import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
-  const [color, setColor] = useState(false);
-
-  const handleClick = () => {
-    if (color === true) {
-      setColor(false)
-      props.decrementLikes()
-    } else if (color === false) {
-      setColor(true)
-      props.incrementLikes()
-    }
+  const handleClick = (event) => {
+    props.onLikeClicked(props.id)
   }
```

Comment:
👍 This approach lets us simplify ChatEntry by having it focus solely on notifier some other code (whatever was passed in as the `onLikeClicked` callback) that some message's like button was clicked. But it makes no assumptions at all about how to handle that event. This is an important way to structure UI code, focusing solely on the notification of the event, and not on what should happen in response to the event.